gpytorch/lazy/kronecker_product_lazy_tensor.py

```diff
@@ -149,6 +150,33 @@ def _size(self):
         right_size = _prod(lazy_tensor.size(-1) for lazy_tensor in self.lazy_tensors)
         return Size((*self.lazy_tensors[0].batch_shape, left_size, right_size))
 
+    @cached(name="svd")
+    def _svd(self) -> Tuple[LazyTensor, Tensor, LazyTensor]:
+        U, S, V = [], [], []
+        for lt in self.lazy_tensors:
+            U_, S_, V_ = lt.svd()
+            U.append(U_)
+            S.append(S_)
+            V.append(V_)
+        S = KroneckerProductLazyTensor(*[DiagLazyTensor(S_) for S_ in S]).diag()
+        U = KroneckerProductLazyTensor(*U)
+        V = KroneckerProductLazyTensor(*V)
+        return U, S, V
+
+    @cached(name="symeig")
+    def _symeig(self, eigenvectors: bool = False) -> Tuple[Tensor, Optional[LazyTensor]]:
+        evals, evecs = [], []
```

Comment:
If we wanted to sort here, we could sort evals and then use like a `PermutationLazyTensor` to represent shuffling the eigenvectors.

Reply:
yeah - not sure if we do want to sort though. Consistency w/ pytorch methods would be nice but if we never use that assumption then why do the extra work?